src/ompl/tools/multiplan/src/ParallelPlan.cpp

```diff
@@ -172,8 +174,10 @@ void ompl::tools::ParallelPlan::solveMore(base::Planner *planner, std::size_t mi
         unsigned int nrSol = ++foundSolCount_;
         foundSolCountLock_.unlock();
 
-        if (nrSol >= maxSolCount)
-            ptc->terminate();
+        if (nrSol >= maxSolCount) {
```

Comment:
let's undo this change, it just makes keeping up-to-date with the upstream more difficult